src/instruments/thorlabs/_abstract.py

```diff
@@ -84,6 +84,9 @@ def querypacket(self, packet, expect=None, timeout=None, expect_data_len=None):
             if resp or timeout is None:
                 break
             else:
+                # Prevent multiple reads before time.time() updates
+                # https://github.com/instrumentkit/InstrumentKit/issues/375
+                time.sleep(1e-32)
                 tic = time.time()
                 if tic - t_start > timeout:
                     break
```

Comment:
As mentioned in the issue, I don't like this fix, since we should rather fix the test. The minimum number of times that `time.time()` should be called is twice. Since on a fast machine I can see that `tic - t_start` can end up being 0, thus it wouldn't break. Can you simply loosen the condition in the test to say: 

``python
assert time_spy.call_count >= 2
``

This should make the test pass, while not modifying the driver and adding an unnecessary call to `time.sleep()`, since this is purely a call for the test.

Reply:
That's a better idea, my solution did feel gross.

Reply:
There we are, a nice 1 character fix. Should be good now.